labs/iot/gps-lab/README.md

```diff
@@ -0,0 +1,53 @@
+# GPS Lab
+
+This folder contains a lab which displays the GPS coordinate in an Azure Map in real time. It collects the GPS coordinates from a GPS sensor attached to a Raspberry Pi and transmits the data
```

Comment:
The product is Azure Maps (maps plural) - can you ensure it's always referred to as Azure Maps as opposed to Azure Map.